brave-lists/debounce.json

```diff
@@ -375,6 +375,16 @@
     "action": "regex-path",
     "param": "^/c/s/(.*)$"
   },
+  {
+    "include": [
+      "*://streaklinks.com/*"
+    ],
+    "exclude": [
+    ],
+    "prepend_scheme": "https",
+    "action": "regex-path",
+    "param": "^/streaklinks.com/(.*)$"
```

Comment:
This is not going to work because it needs to skip the `BQ_bCtlcnH91IqmN1QFG0nUm` bit. Also, the regular expression is just over the path of the URL. Right now, this is looking for the hostname too.

So I think (I haven't tested it), we need something like: `^/[^/]+/(.*)$`

Reply:
Should this be `^\/[^\/]+\/(.*)$` rather than `^/[^/]+/(.*)$`  @ShivanKaul ?  Not too familar with the regex syntax used here

Reply:
No need to escape the slashes, see this example here: https://github.com/brave/adblock-lists/blob/ab585082db75a2afd6e88874559c0971fb6faaed/brave-lists/debounce.json#L387